src/Deserialization/UnsafeContext.cs

```diff
@@ -0,0 +1,65 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using LLVMSharp.Interop;
+
+namespace Symbolica.Deserialization
+{
+    internal sealed unsafe class UnsafeContext : IUnsafeContext
```

Comment:
Has to be.

Reply:
Is there a more specific name for this than just `UnsafeContext`?

Reply:
Doesn't even have to be one thing really. Also don't actually need this class at all. Could just move the various calls to where they are used and make those methods unsafe instead. I just decided to have them all in one place but I'm not particularly tied to this design.